components/logins/android/src/main/java/mozilla/appservices/logins/DatabaseLoginsStorage.kt

```diff
@@ -207,6 +207,13 @@ class DatabaseLoginsStorage(private val dbPath: String) : AutoCloseable, LoginsS
         }
     }
 
+    @Throws(LoginsStorageException::class)
+    override fun rekeyDatabase(newEncryptionKey: String) {
```

Comment:
We ended up having to add a byte-array version of the other API (which hex encodes). Do we need to do that for this too?

Reply:
Added, also rebased the PR